### ISSUE TITLE
When on the back of the card filling out the cvv, the front bleeds through

### ISSUE THREAD

**Issue by dep** [jessepollak/card#276]
Here's a screenshot:

http://dl.dpeck.net/1aoBG

**Comment by dep** [comment]
This is actually happening on your homepage

http://dl.dpeck.net/cCks

Perhaps something new with Chrome?

**Comment by cravindra** [comment]
+1

**Comment by rodolfobubb** [comment]
+1

**Comment by TommyChanthaboune** [comment]
+1

http://imgur.com/L1qfMHg

Chrome Version 53.0.2785.116 (64-bit)

**Comment by TommyChanthaboune** [comment]
if you add the property ` backface-visibility: hidden;` to the `jp-card` class it seems to fix this issue on chrome.

**Comment by dep** [comment]
@TommyChanthaboune That fixed it for me. thanks!

**Comment by rodolfobubb** [comment]
@TommyChanthaboune Fixed it for me too. thanks!

**Comment by seglavi** [comment]
@TommyChanthaboune thanks. it works. and it works too;

http://stackoverflow.com/a/13323984

if your card color change by card types this works more good for me.

```
         .jp-card .jp-card-front,
         .jp-card .jp-card-back {
	        -webkit-transform:rotateY(0deg);
            -webkit-backface-visibility: hidden;
            backface-visibility: hidden;
            ...
```

thanks.

**Comment by jessepollak** [comment]
Fixed in #285, shipping as `v2.0.4`. Thanks y'all!
